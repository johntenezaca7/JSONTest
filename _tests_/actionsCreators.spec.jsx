import {
  nameInput,
  incrementScore,
  decrementScore,
  restart
} from "../react/actionCreator";

import jest from "jest";

test("nameInput", () => {
  expect(nameInput("John")).toMatchSnapshot();
});

test("incrementScore", () => {
  expect(incrementScore(5)).toMatchSnapshot();
});

test("decrementScore", () => {
  expect(decrementScore(5)).toMatchSnapshot();
});

test("restart", () => {
  expect(restart()).toMatchSnapshot();
});
