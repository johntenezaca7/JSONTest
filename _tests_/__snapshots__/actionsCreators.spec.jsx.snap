// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`decrementScore 1`] = `
Object {
  "payload": 5,
  "type": "DECREMENT",
}
`;

exports[`incrementScore 1`] = `
Object {
  "payload": 5,
  "type": "INCREMENT",
}
`;

exports[`nameInput 1`] = `
Object {
  "payload": "John",
  "type": "NAME_INPUT",
}
`;

exports[`restart 1`] = `
Object {
  "payload": 0,
  "type": "RESTART",
}
`;
